packages/frontend/src/features/post/components/PostList/PostList.tsx

```diff
@@ -126,22 +126,22 @@ export default function PostList() {
         post: PostInfo,
     ): Promise<boolean> => {
         try {
-            if (post.isMine) {
+            if (post.isMine && post.finalAction !== null) {
```

Comment:
Why are users only allowed to cancel a vote on themselves' post?

Reply:
This is for already voted, if already voted, need cancel vote first

Reply:
Why need to check `post.isMine`?

Reply:
isMine mean voted post.

Reply:
OK. I get it.